src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -224,4 +224,10 @@
         "the future. Replace by NUM_CPU on a single compute node.",
         check=lambda line: "NUM_NODES" in line,
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="MEMORY_PER_CPU as QUEUE_OPTION to SLURM is deprecated and will removed in "
+        "the future.",
```

Comment:
We should probably have some resolution recommendation in the message.

Reply:
I added `Use REALIZATION_MEMORY instead`